code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -516,6 +522,35 @@
 		"is stabbing themselves with their mandibles!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/toggle_biolum,
+		)
+
+/mob/living/carbon/human/proc/toggle_biolum()
+	set category = "IC"
+	set name = "Toggle Bioluminescence"
+	set desc = "Toggle your bioluminescence effect, at the cost of hunger"
+
+	var/obj/item/organ/internal/lantern/lantern = get_int_organ(/obj/item/organ/internal/lantern)
+
+	if(!lantern)
+		to_chat(src, "<span class='warning'>You are lacking your bioluminescent lantern!</span>")
+		return
+
+	lantern.toggle_biolum()
+
+/mob/living/carbon/human/proc/force_toggle_biolum()
```

Comment:
IPCs are a subtype of the human path, so they'd have access to this proc for overriding.